x-pack/auditbeat/module/system/host/_meta/data.json

```diff
@@ -0,0 +1,54 @@
+{
+    "@timestamp": "2017-10-12T08:05:34.853Z",
+    "agent": {
+        "hostname": "host.example.com",
+        "name": "host.example.com"
+    },
+    "event": {
+        "action": "host",
+        "dataset": "host",
+        "module": "system",
+        "kind": "state"
+    },
+    "message": "Ubuntu host ubuntu-bionic (IP: 10.0.2.15) is up for 0 days, 5 hours, 11 minutes",
+    "service": {
+        "type": "system"
+    },
+    "system": {
+        "audit": {
+            "host": {
```

Comment:
This is not a blocker for this PR, I think we should merge it sooner than later.

But we'll need to revisit this wrt ECS. Almost all keys under `system.audit.host` map perfectly to ECS at the top level. So they should be moved there.

The exceptions I see are `system.audit.host.uptime` and `system.audit.host.timezone.offset.sec`. And lastly, the `system.audit.host.timezone.name` should become `event.timezone`.

Reply:
The top level `host` object is populated by the `add_host_metadata` processor. That processor is using a cache and so will not report the most up-to-date information. What would happen if we didn't include our own data here is that a change event would not actually have any changed data - it would still report the old state because of the cache.

Reply:
This is a very interesting edge case. It's way outside of the scope of this PR to fix, of course. 

But the end goal with these field names is to have this data in the same fields, because the we want to allow for correlation across indices, and for things to be named as expected, and so on.

The fact that at this stage, the processor would override this is a very real problem that we'll need to work around, however. Not sure what the best course of action would be here. Modify Auditbeat to prevent this from happening? Or modify the processor to consider Auditbeat differently and never override the fields?

cc @ruflin